.eslintrc.json

```diff
@@ -44,15 +44,20 @@
         "natural": true
       }
     ],
-    "sort-vars": "error",
-    "strict": ["error", "global"]
```

Comment:
remove since es6 by default in strict mode and webpack also automatically adds it to the output

Reply:
This is intentional, and is used for configuration files that are still written in CommonJS.